src/main/webapp/scripts/account-info.js

```diff
@@ -12,7 +13,11 @@ function updateAccountData() {
         loggedInStatus = loginData.loggedIn;
         logInOutURL = loginData.logInOutURL;
         signupURL = loginData.signUpURL;
+        if (loggedInStatus) {
+          userInfo = loginData.userInfo;
+          console.log(userInfo);
```

Comment:
Remove - we should avoid putting debugging statements like these in the checked-in code.

Reply:
removed

Reply:
removed